sqlalchemy_crud_plus/crud.py

```diff
@@ -13,6 +13,18 @@
 class CRUDPlus(Generic[Model]):
     def __init__(self, model: Type[Model]):
         self.model = model
+        self.primary_key = self._get_primary_key()
+
+    def _get_primary_key(self):
+        """
+        Dynamically retrieve the primary key column(s) for the model.
+        """
+        mapper = inspect(self.model)
+        primary_key = mapper.primary_key
+        if len(primary_key) == 1:
+            return primary_key[0]
+        else:
+            raise ValueError("Composite primary keys are not supported")
```

Comment:
A custom error class is needed here.

Reply:
I have update the code ,please review ,thanks